README.md

```diff
@@ -286,6 +286,23 @@ For example, this command can be used to compile and flash an Arduino Nano RP204
 
 If you want more information about the `nina-fw` firmware, or want to add support for other ESP32-equipped boards, please see https://github.com/arduino/nina-fw
 
+## CYW43439 (RP2040-W)
+
+Go Bluetooth has bare metal support for boards that include a separate CYW43439 Bluetooth Low Energy radio co-processor.
+
+Currently supported boards include:
+
+* [Raspberry Pi Pico RP2040-W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html#raspberry-pi-pico-w)
+* [Pimoroni Badger2040-W](https://shop.pimoroni.com/products/badger-2040-w)
+
+After you have installed TinyGo and the Go Bluetooth package, you should be able to compile/run code for your device.
+
+For example, this command can be used to compile and flash a Pico RP2040-W board with the example we provide that turns it into a BLE peripheral to act like a heart rate monitor:
+
+	tinygo flash -target pico -tags cyw43439 ./examples/heartrate
```

Comment:
Perhaps it makes sense to add a `pico-w` (or `picow` or whatever) target that is specific to this board? (And of course sets the cyw43439 build tag).

Reply:
that is a very good suggestion. I will make a suitable PR to the main repo.

Reply:
See https://github.com/tinygo-org/tinygo/pull/4291

Reply:
Done.